modules/imaging_browser/templates/imaging_session_controlpanel.tpl

```diff
@@ -1,3 +1,7 @@
+{if $subject.error }
```

Comment:
You could still display parts of the panel here, even though `if $subject.error` is true

Reply:
The left side link will only be displayed if there is anything to display. The QC part has been removed if the session is not configured to avoid to do quality control on a not configured session. Please check if this is the desired effect.